src/client/serialize/daos_serialize.c

```diff
@@ -786,17 +786,22 @@ deserialize_props(daos_handle_t poh, hid_t file_id, daos_prop_t **_prop, uint64_
 		if (rc != 0) {
 			D_GOTO(out, rc);
 		}
-		rc = daos_cont_open(poh, label, DAOS_COO_RW, &coh, &cont_info, NULL);
-		if (rc == -DER_NONEXIST) {
-			/* label doesn't already exist so deserialize */
-			deserialize_label = true;
+		if (label[0]) {
```

Comment:
i guess this goes back to my earlier question as this change might not be necessary if we just don't add the attribute with an empty value.

Reply:
ok. I will try not adding the property if the label is empty to see whether it breaks anything. Thanks!